src/lro/stateMachine.ts

```diff
@@ -133,12 +139,36 @@ export function createGetLROState<TResult>(
     spec,
     config.mode
   );
-  return async (path: string): Promise<LROState<TResult>> => {
+  return async (
+    path: string,
+    pollerConfig: PollerConfig
+  ): Promise<LROState<TResult>> => {
     await poll(path);
+    if (retryAfter !== undefined) {
+      const retryAfterInMs = parseInt(retryAfter);
+      pollerConfig.intervalInMs = isNaN(retryAfterInMs)
+        ? calculatePollingIntervalFromDate(
+            new Date(retryAfter),
```

Comment:
Just double checking if you are passing `retryAfter` instead of `retryAfterInMs` on purpose.

Reply:
If we don't get neither pollingInterval, nor Retry-After. Are we setting a default value?

Reply:
For your first question, yes this is intended because `retryAfterInMs` is unusable since it is `NaN` in this branch.